numpy/polynomial/polyutils.py

```diff
@@ -786,12 +786,11 @@ def _deprecate_as_int(x, desc):
         else:
             if ix == x:
                 warnings.warn(
-                    "In future, this will raise TypeError, as {} will need to "
-                    "be an integer not just an integral float."
-                    .format(desc),
+                    f"In future, this will raise TypeError, as {desc} will "
+                    "need tobe an integer not just an integral float.",
```

Comment:
Typo, "tobe" <- "to be"

Reply:
Always worth running a spell check when working with strings :)

Reply:
Thank Charles, sorry about that one. Good catch.